src/pages/Home/components/Portfolio/Prompt/PromptElements.tsx

```diff
@@ -25,12 +27,57 @@ const Avatar = styled(Stack)(() => ({
   },
 }));
 
-export const AIDialog = ({ message, scrollToBottom }) => {
+export const AIDialog = ({
+  message,
+  scrollToBottom,
+  isDialogOpen,
+}: {
+  message: PromptItem;
+  scrollToBottom: () => void;
+  isDialogOpen?: boolean;
+}) => {
   const theme = useTheme();
   const [isTextTyped, setIsTextTyped] = useState(false);
 
+  const messageLength = useMemo(
+    () => message.content.length,
+    [message.content.length],
+  );
```

Comment:
This seems to be unnecessary. We could just use `message.content.length` down below